src/dyad/common/dyad_rc.h

```diff
@@ -15,20 +15,14 @@
 
 #define DYAD_DLL_HIDDEN __attribute__ ((__visibility__ ("default")))
 
-#if DYAD_PERFFLOW
-#define DYAD_PFA_ANNOTATE __attribute__ ((annotate ("@critical_path()")))
-#else
-#define DYAD_PFA_ANNOTATE
-#endif
-
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 enum dyad_core_return_codes {
-    DYAD_RC_OK = 0,                   // Operation worked correctly
-    DYAD_RC_SYSFAIL = -1,             // Some sys call or C standard
-                                      // library call failed
+    DYAD_RC_OK = 0,        // Operation worked correctly
```

Comment:
We should do the formatting part of different PR.

Reply:
Agree. I will run formatter once all the PRs are merged.